src/global.css

```diff
@@ -18,14 +18,7 @@ body {
   margin: 0;
   padding: 0;
   text-rendering: optimizeLegibility;
-  background-image: linear-gradient(
-    120deg,
-    #eea2a2 0,
-    #bbc1bf 19%,
-    #57c6e1 42%,
-    #b49fda 79%,
-    #7ac5d8 100%
-  );
+  background-color: rgb(12, 4, 4);
```

Comment:
You can also handle globals with styled components: https://styled-components.com/docs/api#createglobalstyle